### PR TITLE
Add loop button icons on the media remote

### PR DESCRIPTION
**Pull request by jdragonbae**
This patch adds loop button icons for the media remote. It is for the feature
which enable media loop. A user can select whether media playing
repeatedly or not.

Refs: #1119

### CODE REVIEW

**Review by codeimpl — COMMENTED** [review]
LGTM

File: res/drawable/ic_loop.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+     License, v. 2.0. If a copy of the MPL was not distributed with this
+     file, You can obtain one at https://mozilla.org/MPL/2.0/. -->
+
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24.0"
+    android:viewportHeight="24.0">
+    <path
+        android:pathData="M0 0h24v24H0V0z" />
```
Comment (romandev) [inline res/drawable/ic_loop.xml:13]: This might be unnecessary

File: res/drawable/ic_loop_on.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+     License, v. 2.0. If a copy of the MPL was not distributed with this
+     file, You can obtain one at https://mozilla.org/MPL/2.0/. -->
+
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24.0"
+    android:viewportHeight="24.0">
+    <path
+        android:pathData="M0 0h24v24H0V0z" />
```
Comment (romandev) [inline res/drawable/ic_loop_on.xml:13]: ditto

**Review by romandev — COMMENTED** [review]
lgtm with nit

File: res/drawable/ic_loop.xml (hunk 1 above)
Comment (jdragonbae) [reply res/drawable/ic_loop.xml:13]: Thanks for the detail. Done

File: res/drawable/ic_loop_on.xml (hunk 2 above)
Comment (jdragonbae) [reply res/drawable/ic_loop_on.xml:13]: Done

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**